src/App.js

```diff
@@ -1,5 +1,9 @@
 import { gql, useQuery } from "@apollo/client";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Register from "./routes/Register";
+import SignIn from './routes/SignIn';
```

Comment:
Consider adding an index file inside the routes dir so it cleans up this import with named exports: 

```suggestion
import { Register, SignIn } from './routes';
```

Reply:
yeah lets do this